src/python_routing_v02/compute_nhd_routing_SingleSeg_v02.py

```diff
@@ -212,10 +270,41 @@ def main():
     param_df = param_df.sort_index()
     param_df = nhd_io.replace_downstreams(param_df, cols["downstream"], 0)
 
-    if args.ql:
-        qlats = nhd_io.read_qlat(args.ql)
+    # STEP 5: Read (or set) QLateral Inputs
+    if showtiming:
+        start_time = time.time()
+    if verbose:
+        print("creating qlateral array ...")
+
+    # initialize qlateral dict
+    qlateral = {}
```

Comment:
I think we can strike the qlateral dictionary... it must be leftover from prior versions.